src/styles/calendar.scss

```diff
@@ -0,0 +1,36 @@
+.calendar {
+  position: relative;
+  width: 800px;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  margin: 0 auto;
+  padding-left: 47px;
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), avoid using `px` as a unit for padding. Consider using relative units like `rem` or `em` for consistency and scalability.